### PR TITLE
Feature 208

### PR DESCRIPTION
**Pull request by TPrevoteau**
## Content

Please describe your contribution here...

---

## Checklist

Please check that your pull request is correct:

- Each commit:
    - [ ] corresponds to a contribution that should be notified to users,
    - [ ] does not generate new errors or warnings at compile or test time,
    - [ ] must be attributed to its real authors (with correct GitHub IDs and [correct syntax for multiple authors](https://help.github.com/articles/creating-a-commit-with-multiple-authors/)).
- The title of a commit should:
    - [ ] begin with a contribution type
        - `FEATURE` for a behaviour allowing a user to do something new,
        - `FIX` for a behaviour which has been changed in order to meet user’s expectations,
        - `TEST` when it concerns an acceptance test,
        - `PROCESS` for a change in the way the software is built, tested, deployed,
        - `DOC` when it concerns only internal documentation (however it is better to combine it with the contribution that required this documentation change),
    - [ ] be followed by a colon (`:`) with one space after and no space before,
    - [ ] be followed by a title (written in English) as short, as user-centered and as explicit as possible
        - If it is a feature, the title must be the user action (beginning with a verb, and please not `manage`),
        - If it is a fix, the title must describe the intended behavior (with `should`).
    - [ ] ends with a reference to the corresponding ticket with the following syntax:
        - `(closes #xx)` if xx is a feature ticket (and the commit is a complete implementation),
        - `(fixes #xx)` if xx is a fix ticket (and the commit is a complete fix),
        - `(see #xx)` otherwise,
- Each committed line is:
    - [ ] useful (it would not work if removed)
        - if it is a comment line, its information could not be conveyed by better variables and function naming, better code structuring, or better commit message,
    - [ ] related to this very contribution (feature, fix...),
    - [ ] in English (with the exception of Gherkin scenarios in French and resulting steps),
    - [ ] without any typo in variable, class or function names,
    - [ ] correctly indented (spaces rather than tabs, same number of characters as in the rest of the file).

### CODE REVIEW

File: src/components/itemPage/Attribute.jsx [hunk 1]
```diff
@@ -89,6 +89,11 @@ function AttributeValue(props) {
       />
     </div>
   );
+  if (props.value.startsWith('http')) return (
```
Comment (benel) [inline src/components/itemPage/Attribute.jsx]: Que se passe-t-il si `props.value` est à null ?

Comment (TPrevoteau) [reply src/components/itemPage/Attribute.jsx]: Je suppose qu'il soit y avoir une erreur du type "Cannot apply method startsWith on undefined"
J'ai apporté la modification nécessaire.

File: src/components/itemPage/Attribute.jsx [hunk 2]
```diff
@@ -89,11 +89,13 @@ function AttributeValue(props) {
       />
     </div>
   );
-  if (props.value.startsWith('http')) return (
-    <a href={props.value} className="Value">
-      {props.value}
-    </a>
-  );
+  if(props.value) {
+    if (props.value.startsWith('http')) return (
```
Comment (benel) [inline src/components/itemPage/Attribute.jsx:93]: Héhé, vous pouvez mettre deux conditions dans le même `if` : `if (a && b`.)

**Comment by benel** [comment]
- À part le petit problème d'esthétique du code, le code est OK... d'ailleurs le test a réussi !!!
- Par contre, pour l'historique vous devez appliquer toutes les règles vues précédemment : 
  - [ ] regrouper vos deux commits en un seul
  - [ ] mettre le préfixe (`FEATURE:`)
  - [ ] l'intitulé de la fonctionnalité (celui qui apparaît dans le ticket)
  - [ ] entre parenthèses : 
    - [ ] le mot-clé `closes` (ainsi le ticket sera fermé automatiquement quand le commit sera intégré à la branche maître,
    - [ ] le numéro du ticket (précédé de `#`) (ce qui créera un lien bidirectionnel entre le commit et le ticket).

Vous voyez comment vous y prendre avec git ?

**Comment by benel** [comment]
Vu.

**Pull request closed without merging**